llvm/test/MC/X86/apx/crc32-att.s

```diff
@@ -0,0 +1,73 @@
+# RUN: llvm-mc -triple x86_64 --show-encoding %s | FileCheck %s
+
+# CHECK: crc32b	%r16b, %r22d
```

Comment:
Missing test for code like `{evex} crc32b %eax, %ebx`

Reply:
OK, then we also miss test for other promoted instrs, I just test egpr for them.